src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -191,7 +194,9 @@ class RunWorkflowTests : SeleniumTest()
         createButton.click()
         wait.until(ExpectedConditions.presenceOfElementLocated(By.id("git-branch")))
 
-        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isEqualTo(1)
+        // There should be only one git commit option, but bk needs 2 to run successfully; possibly related to running a workflow
```

Comment:
```suggestion
        // We generally expect one git commit option in demo orderly before refresh, but have found that this can be two commits on Buildkite run. This may be related to having run a workflow in another test
```
might be a little clearer

Also - I think there's an outstanding comment from me to check for no errors displayed following refresh git, since the number of options shown (checked on line 203) is now a bit less meaningful.

Reply:
I put a check for the error component in, but according to Mark's comment above, it'll never be rendered so I don't know what else can be checked in a Selenium test.